src/lib/types/experimental.ts

```diff
@@ -310,17 +310,10 @@ const flags: IFlags = {
         process.env.UNLEASH_EXPERIMENTAL_DATA_USAGE_MULTI_MONTH_VIEW,
         false,
     ),
-    uiGlobalFontSize: {
-        name: 'uiGlobalFontSize',
-        enabled: parseEnvVarBoolean(
-            process.env.EXPERIMENTAL_UI_GLOBAL_FONT_SIZE_NAME,
-            false,
-        ),
-        payload: {
-            type: PayloadType.JSON,
-            value: '14',
-        },
-    },
+    uiGlobalFontSize: parseEnvVarBoolean(
+        process.env.EXPERIMENTAL_UI_GLOBAL_FONT_SIZE_NAME,
+        false,
+    ),
```

Comment:
Didn't you do this already in https://github.com/Unleash/unleash/pull/9290. Is your branch up to date?